packages/crew_mvdir.rb

```diff
@@ -0,0 +1,35 @@
+require 'package'
+
+class Crew_mvdir < Package
+  description 'Alternative for "rsync --remove-source-files dir1/ dir2/", with faster speed'
```

Comment:
`Faster alternative for "rsync --remove-source-files dir1/ dir2/"`?

Reply:
Added "Faster" to description.